code/modules/atmospherics/machinery/components/unary_devices/vent_pump.dm

```diff
@@ -31,9 +31,25 @@
 	var/datum/radio_frequency/radio_connection
 	var/radio_filter_out
 	var/radio_filter_in
+	var/list/motors
+	var/max_motors = 5
+	var/transfer_modifier = 1
 
 	pipe_state = "uvent"
 
+/obj/machinery/atmospherics/components/unary/vent_pump/attackby(obj/item/I, mob/user, params)
+	if(istype(I, /obj/item/assembly/motor))
+		var/obj/item/assembly/motor/M = I
+		if(LAZYLEN(motors) > max_motors)
+			to_chat(user, "<span class='warning'>There are too many motors here!</span>")
+			return
+		to_chat(user, "<span class='notice'>You start mounting a motor.</span>")
+		if(do_after(user, 1.5 SECONDS, target = src))
+			to_chat(user, "<span class='notice'>You mounted a motor and the fans are spinning faster now.</span>")
+			LAZYADD(motors, M)
+			transfer_modifier *= 1000
```

Comment:
These values are insane. 5 of these would amount to 10^15, which based on how this is used is absolutely bonkers. You should consider making this additive.

Reply:
ok so i know that the values are high and the whole thing is wonky at best, broken at worst. The thing is that i made those modifications for a quick test to see if/how they worked, i didnt do much on those because i'm focusing on the fusion reaction side. I'll refine this both code and balance wise later on in the PR.

Reply:
Ahhh that makes total sense. I understand